src/program.fs

```diff
@@ -143,6 +151,8 @@ module Program =
           onError = program.onError
           termination = mapTermination program.termination }
 
+    module Subs = Subs.Internal
```

Comment:
I believe this is re-exporting the "internal" module as "public" `Subs` module, maybe it should just open?

Reply:
It's a module alias, stays local to the file. I can use an open if preferred. But didn't want to use `stop` and `change` unprefixed. And Internal was too long of a prefix for me.